Tests/configure_and_test_integration_instances.py

```diff
@@ -1313,23 +1380,28 @@ def main():
                 installed_content_packs_successfully = install_packs(build, pack_ids=pack_ids)
     else:
         installed_content_packs_successfully = True
+    return installed_content_packs_successfully
+
+
+def main():
+    install_logging('Install Content And Configure Integrations On Server.log')
+    build = Build(options_handler())
+
+    configure_servers_and_restart(build)
+    installed_content_packs_successfully = install_packs_pre_update(build)
 
-    tests_for_iteration = get_tests(build)
     new_integrations, modified_integrations = get_changed_integrations(build)
-    all_module_instances, brand_new_integrations = configure_server_instances(build,
-                                                                              tests_for_iteration,
-                                                                              new_integrations,
-                                                                              modified_integrations)
-    successful_tests_pre, failed_tests_pre = instance_testing(build, all_module_instances, pre_update=True)
-    if LooseVersion(build.server_numeric_version) < LooseVersion('6.0.0'):
-        update_content_till_v6(build)
-    elif not build.is_nightly:
-        set_marketplace_url(build.servers, build.branch_name, build.ci_build_number)
-        installed_content_packs_successfully = install_packs(build) and installed_content_packs_successfully
 
-    all_module_instances.extend(brand_new_integrations)
-    successful_tests_post, failed_tests_post = instance_testing(build, all_module_instances, pre_update=False)
-    disable_instances(build, all_module_instances)
+    pre_update_configuration_results = configure_and_test_integrations_pre_update(build,
+                                                                                  new_integrations,
+                                                                                  modified_integrations)
+    old_module_instances, new_module_instances, failed_tests_pre, successful_tests_pre = pre_update_configuration_results
+    installed_content_packs_successfully = update_content_on_servers(build, installed_content_packs_successfully)
```

Comment:
no reason to pass `installed_content_packs_successfully` to the function
```suggestion
    installed_content_packs_successfully = update_content_on_servers(build) and installed_content_packs_successfully
```